src/main/java/stream/flarebot/flarebot/commands/general/ReportsCommand.java

```diff
@@ -0,0 +1,232 @@
+                            errorBuilder.setDescription("Invalid status: " + args[2]);
+                            errorBuilder.addField("Statuses", "```\nOPEN\nON_HOLD\nRESOLVED\nCLOSED\nUNDER_REVIEW\nDUPLICATE\n```", false);
+                            MessageUtils.sendErrorMessage(errorBuilder, channel);
+                            return;
+                        }
+                        Report report = ReportManager.getReport(channel.getGuild().getId(), id);
+                        report.setStatus(status);
+
+                        ReportManager.reportsToSave.add(report);
+                    } else {
+
+                    }
+                }
+                break;
+                case "report": {
+                    User user = MessageUtils.getUser(args[1]);
+                    if(user == null){
+                        MessageUtils.sendErrorMessage("Invalid user: " + args[1], channel);
+                        return;
+                    }
+
+                    String reason = MessageUtils.getMessage(args, 2);
+
+                    Report report = new Report(channel.getGuild().getId(), 0, reason, sender.getId(), user.getId(), new Timestamp(System.currentTimeMillis()), ReportStatus.OPEN);
+
+                    ReportManager.reportsToSave.add(report);
+                }
+                break;
+                default: {
+                    MessageUtils.sendUsage(this, channel);
+                }
+            }
+        } else {
+            MessageUtils.sendUsage(this, channel);
+        }
+    }
+
+    private String getReportsTable(Guild guild, List<Report> reports) {
+        ArrayList<String> header = new ArrayList<>();
+        header.add("Id");
+        header.add("Reporter");
+        header.add("Reported");
+        header.add("Time");
+        header.add("Status");
+
+        List<List<String>> table = new ArrayList<>();
+        for (Report report : reports) {
+            ArrayList<String> row = new ArrayList<>();
+            row.add(String.valueOf(report.getId()));
+            row.add(MessageUtils.getTag(FlareBot.getInstance().getUserByID(String.valueOf(report.getReporterId()))));
+            row.add(MessageUtils.getTag(FlareBot.getInstance().getUserByID(String.valueOf(report.getReportedId()))));
+
+            DateFormat formatedDate = new SimpleDateFormat("MM/dd/yyyy HH:mm"); //US format
+            String date = formatedDate.format(report.getTime());
+
+            row.add(date);
+
+            row.add(report.getStatus().getMessage(guild.getId()));
+
+            table.add(row);
+        }
+
+        return MessageUtils.makeAsciiTable(header, table, null);
+    }
+
+    @Override
+    public String getCommand() {
+        return "reports";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Used to view reports and to report users";
+    }
+
+    @Override
+    public String getUsage() {
+        return "{%}reports\n" +
```

Comment:
Please add descriptions to each line of description. For reference see [this](https://github.com/binaryoverload/FlareBot/blob/dev/src/main/java/stream/flarebot/flarebot/commands/administrator/AutoAssignCommand.java#L122)